### PR TITLE
[flutter_conductor] port codesign script to conductor

### PR DESCRIPTION
**Pull request by christopherfujino**

### CODE REVIEW

File: dev/conductor/core/lib/src/codesign.dart [hunk 1]
```diff
@@ -320,96 +390,789 @@ class CodesignCommand extends Command<void> {
...
+  /// Unzip an [EmbeddedZip] and visit its children.
+  ///
+  /// The [parent] directory is scoped to the parent zip file.
+  @override
+  Future<void> visitEmbeddedZip(EmbeddedZip file, Directory parent) async {
+    final File localFile = await _validateFileExists(file, parent);
+    final Directory newDir = tempDir.childDirectory('embedded_zip_$nextId');
+    final package_arch.Archive? archive = await _unzip(localFile, newDir);
+    if (archive != null) {
+      await _hashActualFiles(archive, newDir);
```
Comment (XilaiZhang) [inline dev/conductor/core/lib/src/codesign.dart:659]: hashActualFiles seem to only be run when archive isn't null, which means the download has failed. So if the download succeeds, then the actual hashes will not be recorded

Comment (christopherfujino) [reply dev/conductor/core/lib/src/codesign.dart:681]: Thanks!

Comment (XilaiZhang) [reply dev/conductor/core/lib/src/codesign.dart:681]: Christopher is too polite QAQ

**Comment by christopherfujino** [comment]
Closing this in favor of https://github.com/flutter/cocoon/pull/1861

**Pull request closed without merging**